registrations/templates/registrations/base.html

```diff
@@ -7,125 +7,34 @@
-      {% block toppanel-text %}
-        {% if request.user.is_impersonate %}
-          <span class="icon-warning">⚠</span>
-          Impersonating user -
-          <a href="{% url 'impersonate-stop' %}" class="uu-red-text">Stop</a>
-          <span class="icon-warning">⚠</span>
-        {% elif acceptation %}
-          <span class="icon-warning">⚠</span>
-          Acceptation server
-          <span class="icon-warning">⚠</span>
-        {% elif debug %}
-          <span class="icon-warning">⚠</span>
-          Local development server
-          <span class="icon-warning">⚠</span>
-        {% endif %}
-      {% endblock %}
-    </div>
-    {% include 'base/language_box.html' %}
-  </div>
-  <div class="uu-header-row">
-    <div class="me-auto uu-header-title">
-      <a href="{% url "main:home" %}">
-        {% block site_header %}
-          {% trans "site:header:title" %}
-        {% endblock %}
-      </a>
-    </div>
-    <div class="ms-auto">
-      {% block login_header %}
-        {% trans "site:header:login_text" %}
-      {% endblock %}
-    </div>
-  </div>
 {% endblock %}
 
-
-{% block uu-content %}
-
-
-  <style>
-  .uu-cover {
-    --bs-uu-cover-default-height: 250px;
-  }
-  </style>
-  {% block cover %}
-    <div class="uu-cover">
-      <img class="uu-cover-image" alt="cover" src="{% static 'procreg/img/ivar-pel-header.jpg' %}" />
-      <div class="uu-cover-copyright">
-        {% block cover_attribution %}Cover by Ivar Pel{% endblock %}
-      </div>
-    </div>
-  {% endblock %}
-
-  {% block hero %}
-    <header class="uu-hero">
-      <h1>
-        {% block hero_text %}
-          {% trans "site:hero:default_hero_text" %}
-        {% endblock %}
-      </h1>
-    </header>
-  {% endblock %}
-  <style>
-  .uu-sidebar-content {
-    height: 100%;
-  }
-    </style>
-  <div class="uu-sidebar-container uu-sidebar-right">
-
-      <aside class="uu-sidebar">
-        <button class="uu-sidebar-toggle" type="button" data-bs-toggle="collapse" data-bs-target="#exampleSidebar" aria-expanded="false">
-          Show sidebar
-        </button>
-        <div id="exampleSidebar" class="collapse uu-sidebar-collapse">
-          {% block sidebar %}
-            <h2>Sidebar</h2>
-          {% endblock %}
+{% block content %}
+    <div class="uu-sidebar-container uu-sidebar-right">
+        <aside class="uu-sidebar">
+            <button class="uu-sidebar-toggle" type="button" data-bs-toggle="collapse" data-bs-target="#exampleSidebar"
+                    aria-expanded="false">
+                Show sidebar
+            </button>
+            <div id="exampleSidebar" class="collapse uu-sidebar-collapse">
+                {% block sidebar %}
+                    <h2>Sidebar</h2>
+                {% endblock %}
+            </div>
+        </aside>
+
+        <div class="uu-sidebar-content">
+            {# We have a lot of blocks with 'content' in the name now; #}
+            {# Is there a better name for this? sidebar-content? #}
```

Comment:
> Flagship - Destroyer - Frigate

Flagships are not inherently 'bigger' than the others; in fact, the Royal Dutch Navy has frigates with flagship capabilities. (Although its frigates are larger than destroyers in other navies, for some reason).

For full silliness, I'd recommend the ruski naming scheme, with western equivalents in brackets:
- Cruiser (Carrier), Cruiser (Battleship), Cruiser (Cruiser), Cruiser (Destroyer), Cruiser (Frigate), Guard Ship (Corvette)

(Almost) everything is a cruiser! Why? Because Turkey.


Jokes aside, I have some real problems with naming stuff in the DSC templates that I cannot fix without a major release. So dunno if following DSC at the moment is the best idea atm